src/metabase/driver/util.clj

```diff
@@ -24,23 +24,26 @@
   "Generic error messages that drivers should return in their implementation
   of [[metabase.driver/humanize-connection-error-message]]."
   {:cannot-connect-check-host-and-port
-   {:message [(deferred-tru "Hmm, we couldn''t connect to the database.")
-              " "
-              (deferred-tru "Make sure your Host and Port settings are correct")]
+   {:message (deferred-tru
+              (str "Hmm, we couldn''t connect to the database."
+                   " "
+                   "Make sure your Host and Port settings are correct"))
     :errors  {:host (deferred-tru "check your host settings")
               :port (deferred-tru "check your port settings")}}
 
    :ssh-tunnel-auth-fail
-   {:message [(deferred-tru "We couldn''t connect to the SSH tunnel host.")
-              " "
-              (deferred-tru "Check the Username and Password.")]
+   {:message (deferred-tru
+              (str "We couldn''t connect to the SSH tunnel host."
```

Comment:
I like the use of the multiline translation string but i'm not seeing the point of it here.

Reply:
Originally, these were multiline translation strings. When I migrated them I implemented a hack to make them work at runtime. Now, after having discussed with Noah, I've migrated them to the "canonical" multiline form.
Does this explanation make sense to you?